src/renderer/components/Invitations/index.vue

```diff
@@ -86,7 +86,7 @@ export const metadata = {
     group: 'Agent to Agent',
     priority: 10,
     required_protocols: [
-      'did:sov:BzCbsNYhMrjHiqZDTUASHg;spec/admin-connections/0.1'
+      'https://github.com/hyperledger/aries-toolbox/tree/master/docs/admin-connections/0.1'
```

Comment:
This line needs to be reverted in order to maintain the old backwards compatibility. (We should deprecate this old module but this should be done in a separate PR so we just want to revert for now)